core/src/test/java/apoc/date/DateTest.java

```diff
@@ -135,41 +134,6 @@ public void testToUnixtimeWithEmptyInput() throws Exception {
 				row -> assertEquals(null, row.get("value")));
 	}
 
-	@Test public void testParseAsZonedDateTimeWithCorrectFormat() throws Exception {
-		testCall(db,
-				"RETURN apoc.date.parseAsZonedDateTime('03/23/1965 00:00:00','MM/dd/yyyy HH:mm:ss','America/New_York') AS value",
-				row -> assertEquals(ZonedDateTime.of(LocalDateTime.of(1965, 3, 23, 0, 0), ZoneId.of("America/New_York")),
-						row.get("value")));
-	}
-
-	@Test public void testParseAsZonedDateTimeWithDefaultTimezone() throws Exception {
-		testCall(db,
-				"RETURN apoc.date.parseAsZonedDateTime('03/23/1965 00:00:00','MM/dd/yyyy HH:mm:ss') AS value",
-				row -> assertEquals(ZonedDateTime.of(LocalDateTime.of(1965, 3, 23, 0, 0), ZoneId.of("UTC")),
-						row.get("value")));
-	}
-
-	@Test public void testParseAsZonedDateTimeWithDefaultFormatAndTimezone() throws Exception {
-		testCall(db,
-				"RETURN apoc.date.parseAsZonedDateTime('1965-03-23 00:00:00') AS value",
-				row -> assertEquals(ZonedDateTime.of(LocalDateTime.of(1965, 3, 23, 0, 0), ZoneId.of("UTC")),
-						row.get("value")));
-	}
-
-	@Test public void testParseAsZonedDateTimeWithIncorrectPatternFormat() throws Exception {
-		expected.expect(instanceOf(QueryExecutionException.class));
-		testCall(db,
-				"RETURN apoc.date.parseAsZonedDateTime('03/23/1965 00:00:00','MM/dd/yyyy HH:mm:ss/neo4j','America/New_York') AS value",
-				row -> assertEquals(ZonedDateTime.of(LocalDateTime.of(1965, 3, 23, 0, 0), ZoneId.of("America/New_York")),
-						row.get("value")));
-	}
-
-	@Test public void testToZonedDateTimeWithNullInput() throws Exception {
-		testCall(db,
-				"RETURN apoc.date.parseAsZonedDateTime(NULL) AS value",
-				row -> assertNull(row.get("value")));
-	}
-
```

Comment:
It seems like there are no tests for the replacement apoc.temporal.toZonedTemporal. Would it be worth keeping these types of tests and adapting them to use apoc.temporal.toZonedTemporal?